### ISSUE TITLE
VoiceOver reads content of the status bar when doing navigation

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#87664]
1. Have the following content in your editor. Make sure to have the status bar visible
2. Turn on VoiceOver, navigate to start on second line and start going left / right
3. VoiceOver is prouncing the content of the status bar which it should not 🐛 

```
hello world
i will continue typing more things.
```

Assigning to this milestone to check if this gets fixed with latest electron, or if status bar items are doing something special (they should not).

**Comment by isidorn** [comment]
I just verified this is fixed in the latest VS Code exploration build which uses electron 7.15. Due to that it seems to be an electron issue.

@ivanfetch as a workaround please either use the VS Code exploration build https://az764295.vo.msecnd.net/exploration/25d09f2ef5a379e8a3aedd1c821bd866a5efe58c/VSCode-darwin-exploration.zip or hide the status bar (if that works for you) via F1 > Toggle Status Bar Visibility

As for what causes the current issue: I have verified that the status bar items do not call `alert` and that the only special thing for the status bar is that it uses the [`role=contentinfo`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role). However even after I remove the role I can still reproduce this issue with the current electron 6. So I am quite puzzled as to why Voice Over would read the content of the status bar all of a sudden. Unless status bar does more special things which I am unaware of @bpasero

@deepak1556 what is approximate plan wrt electron 7? Do we plan to ship it in the near future, are there a lot of blockers?

**Comment by ivanfetch** [comment]
@isidorn Hiding the status bar avoids the status bar reading its content, however moving by character does not read the current character. Right-arrow reads nothing, and left-arrow reads the last two characters together. For example, if my cursor is on the letter `d` of the word `world`, and I press left-arrow, I here: ld, rl, or, wo

I am still running my original version and have not yet upgraded to the exploration build.

**Comment by isidorn** [comment]
@ivanfetch thanks for commenting back. First of all this issue should be fixed with altest vscode insiders since we updated to Electron 7
https://code.visualstudio.com/insiders/
If you still see issues with vscode insiders we can discuss further during our sync tonight. Since I also can not reproduce the `world` example you just mentioned.

**Comment by deepak1556** [comment]
@isidorn can we close this, now that the electron update has made into `1.43` ?

**Comment by isidorn** [comment]
@deepak1556 yes, we should. Thanks for the ping.
